content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -102,23 +121,6 @@ To enable reporting of trace data with [Zipkin](https://zipkin.io/), use the `cs
   >__Note__:  
   > - This command will deploy the Authorization Proxy Server in the namespace specified in the input YAML file.
 
-5. Create the `karavi-auth-tls` secret using your own certificate or by using a self-signed certificate generated via cert-manager. 
```

Comment:
Step 4 is creating the karavi-storage-secret, right? That can be removed because we don't need to do that anymore.

Reply:
I've added a note to indicate that the `karavi-storage-secret` is no longer needed for v2.0 Tech Preview.

For all changes in Operator installation, I've tried to avoid duplicating the page since most of the installation follows a similar flow.